packages/compass/release/commands.js

```diff
@@ -143,41 +134,8 @@ async function releaseCheckout(versionLike) {
   cli.info('Switched to branch:', chalk.bold(await git.getCurrentBranch()));
 }
 
-async function releaseChangelog(optionalVersionToCompareTo) {
-  await getValidReleaseBranch();
-  const releaseVersion = await getPackageJsonVersion();
-  await changelog.render(releaseVersion, optionalVersionToCompareTo);
-}
-
-async function releaseWait() {
-  await getValidReleaseBranch();
-  const releaseVersion = await getPackageJsonVersion();
-
-  const evergreenProject = version.isGa(releaseVersion) ?
-    '10gen-compass-stable' : '10gen-compass-testing';
-
-  cli.info(
-    `Waiting ${chalk.bold(releaseVersion)} assets for being built in evergreen:`,
-    ux.link(`https://evergreen.mongodb.com/waterfall/${evergreenProject}`),
-    '\n'
-  );
-
-  await waitForAssets(releaseVersion, {
-    downloadCenter: createDownloadCenter()
-  });
-}
-
 module.exports = {
   releaseBeta,
   releaseGa,
-  releaseCheckout,
-  releaseChangelog,
-  releaseWait
+  releaseCheckout
 };
-
-function createDownloadCenter() {
-  return new CompassDownloadCenter({
-    accessKeyId: env.requireEnvVar('MONGODB_DOWNLOADS_AWS_ACCESS_KEY_ID'),
-    secretAccessKey: env.requireEnvVar('MONGODB_DOWNLOADS_AWS_SECRET_ACCESS_KEY')
```

Comment:
Worth highlighting: if you have these credentials on your machine, after this change (and recent changes to the release flow) you don't need them anymore